django_ai_assistant/helpers/assistants.py

```diff
@@ -422,7 +450,36 @@ class AgentState(TypedDict):
             output: str
 
         def setup(state: AgentState):
-            return {"messages": [SystemMessage(content=self.get_instructions())]}
+            messages = [SystemMessage(content=self.get_instructions())]
+
+            if self.structured_output:
+                schema = None
+
+                # If Pydantic
+                if inspect.isclass(self.structured_output) and issubclass(
+                    self.structured_output, BaseModel
+                ):
+                    schema = json.dumps(self.structured_output.model_json_schema())
+
+                schema_information = ""
+                if schema:
+                    schema_information = f"JSON will have the following schema:\n\n{schema}\n\n"
```

Comment:
will have -> must have

probably better

Reply:
At this point we are just signalizing to the AI the format, in some tests making this prompt more "strict" caused the assistant to enter a loop overusing tools until it had all the information. So this is here more as a hint than an order.